app/controllers/cars_controller.rb

```diff
@@ -1,13 +1,19 @@
 class CarsController < ApplicationController
     # skip_before_action :verify_authenticity_token
+    # CODE_COMMENT: I see that most of the controller / view code you have written is in cars any reason for this? 
```

Comment:
Since we're dealing mostly with cars as the most concentrated topic, it's the one that has most of the code. Apart from this other aspects of the application that important also have a controller.